SafeguardSessions/Src/SafeguardSessions.pq

```diff
@@ -166,6 +166,11 @@ shared HandleIfThereIsAnError = (input as record) =>
         input;
 
 SafeguardSessions = [
+    TestConnection = (dataSourcePath) =>
```

Comment:
We should test the TestConnection method as well.
Have you talked to Szabi about what do we want to consider a connection test? I'm asking because of the null parameters.

Reply:
A thread has been opened on teams but I can't add you to it (A-Team & PM). There are a lot of issues on the service side if we use a lot of optional parameters.